include/rangers-api/ApplicationCommon/Reflection/RangeAttribute.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+namespace app_cmn::rfl {
+    template<typename T>
+    struct alignas(8) RangeAttribute {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.